app/helpers/apply_job_helper.rb

```diff
@@ -1,11 +1,24 @@
+# frozen_string_literal:true
+
+# apply job helper contain methods to verify a user applied for job or not 
 module ApplyJobHelper
-    def applied?(user_id)
-        @apply_job = ApplyJob.find_by(user_id:user_id,job_id:params[:id].to_i)
-        if @apply_job.nil?
-            return false
-        else
-            return true
-        end
+  # Return a user already applied for job or not
+  def applied?(user_id)
+    @apply_job = ApplyJob.find_by(user_id: user_id, job_id: params[:id].to_i)
+    if @apply_job.nil?
+      false
+    else
+      true
```

Comment:
Change this into ---



return false if @apply_job.nil?

true

Reply:
done